### DISCUSSION TITLE
Using general area rather than actual address

### DISCUSSION THREAD

**Discussion by rpjonjubilee** [code4recovery/12-step-meeting-list#882] [Q&A]
I use TSML for a non-AA 12-step fellowship. Most of our groups do not want to list the actual address of the meeting. In the past, I've been able to just provide the general area, like "St. Louis, MO" rather than the exact address. The meetings I created in the past in this way still display properly in TSML, but if I attempt to edit one of these, I get an error "In person meetings must have a specific address." and can not save the updates.

Is there a way to disable this requirement? Any work arounds?

Thanks
Richard H.

**Comment by kiyote33** [comment]
Hi @rpjonjubilee, With the advent of the recent pandemic, we were forced to modify the behavior of our software to handle online meetings, in-person meetings, inactive meetings, and a combination of them which we refer to as "hybrid meetings".  The requirement for in-person meetings to have an actual address is there to ensure all in-person meetings can be found. One option I've seen to bypass this rule is to use a common location address with a location note stating **"** IMPORTANT ** The physical address listed is for administration purposes ONLY. No in-person meetings are actually held at this location.**"
Hope this helps...
